e2e/sendSheetFlow.spec.js

```diff
@@ -0,0 +1,255 @@
+  //   await Helpers.delay(3000);
+  //   await Helpers.checkIfVisible('backup-sheet');
+  //   await Helpers.tap('backup-sheet-imported-cancel-button');
+  // });
+
+  it('Should show all wallet sections', async () => {
+    await Helpers.delay(1000);
+    await Helpers.checkIfElementByTextIsVisible('Investments');
+    await Helpers.swipe('wallet-screen', 'up');
+    await Helpers.checkIfElementByTextIsVisible('Collectibles');
+  });
+
+  it('Should say "poopcoin.eth" in the Profile Screen header', async () => {
+    await Helpers.delay(1000);
+    await Helpers.swipe('wallet-screen', 'right');
+    await Helpers.delay(2000);
+    await Helpers.checkIfElementByTextIsVisible('poopcoin.eth');
+    await Helpers.swipe('profile-screen', 'left');
+  });
+
+  it('Should open send sheet after tapping send fab', async () => {
+    await Helpers.delay(1000);
+    await Helpers.tap('send-fab');
+    await Helpers.delay(1000);
+    await Helpers.checkIfElementByTextIsVisible('To:');
+  });
+
+  it('Should do nothing on typing jibberish send address', async () => {
+    await Helpers.delay(1000);
+    await Helpers.checkIfVisible('send-asset-form-field');
+    await Helpers.typeText('send-asset-form-field', 'gvuabefhiwdnomks', false);
+    await Helpers.delay(1000);
+    await Helpers.checkIfVisible('paste-address-button');
+  });
+
+  it('Should show show Contact Button & Asset List on valid public address', async () => {
+    await Helpers.clearField('send-asset-form-field');
+    await Helpers.delay(1000);
+    await Helpers.checkIfVisible('send-asset-form-field');
+    await Helpers.typeText(
+      'send-asset-form-field',
+      '0xF0f21ab2012731542731df194cfF6c77d29cB31A',
+      false
+    );
+    await Helpers.delay(1000);
+    await Helpers.checkIfVisible('add-contact-button');
+    await Helpers.checkIfVisible('send-asset-list');
+  });
+
+  it('Should show show Contact Button & Asset List on valid ENS address', async () => {
+    await Helpers.clearField('send-asset-form-field');
+    await Helpers.delay(1000);
+    await Helpers.checkIfVisible('send-asset-form-field');
+    await Helpers.typeText('send-asset-form-field', 'poopcoin.eth', false);
+    await Helpers.checkIfVisible('add-contact-button');
+    await Helpers.checkIfVisible('send-asset-list');
+  });
+
+  it('Should display Asset Form after tapping on savings asset', async () => {
+    await Helpers.delay(1000);
+    await Helpers.checkIfVisible('send-savings-cSAI');
+    await Helpers.tap('send-savings-cSAI');
+    await Helpers.delay(1000);
+    await Helpers.checkIfVisible('selected-asset-field-input');
+  });
+
+  it('Should go back to Asset List after tapping on savings asset', async () => {
+    await Helpers.delay(1000);
+    await Helpers.tap('send-asset-form-cSAI');
+    await Helpers.delay(1000);
+    await Helpers.checkIfVisible('send-asset-list');
+  });
+
+  it('Should display Asset Form after tapping on asset', async () => {
```

Comment:
It's very likely that the socks in the wallet will be gone and I'd break the test.
so I'd use DAI  or USDC since we have some most of the time